src/Aspire.Cli/PackageChannels/PackageChannelService.cs

```diff
@@ -0,0 +1,80 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Aspire.Cli.PackageChannels;
+
+/// <summary>
+/// Implementation of package channel service that provides access to predefined and dynamic package channels.
+/// </summary>
+internal sealed class PackageChannelService : IPackageChannelService
+{
+    private static readonly PackageChannel[] s_predefinedChannels =
```

Comment:
@captainsafia This might be a place to plug in org specific feeds if we make it extensible in the future.